package/MDAnalysis/analysis/pca.py

```diff
@@ -308,14 +316,15 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
             Iteration stops *before* this frame number, which means that the
             trajectory would be read until the end.
         step : int, optional
-            Number of frames to skip over for PCA transform. If set to ``None``
+            Include every `step` frames in the PCA transform. If set to ``None``
             (the default) then every frame is analyzed (i.e., same as
             ``step=1``).
```

Comment:
```suggestion
            Include every `step` frames in the PCA transform. If set to
            ``None`` (the default) then every frame is analyzed (i.e., same as
            ``step=1``).
```

I'm running out weird movie-based PEP8 messages to throw.